pyke/kepconvert.py

```diff
@@ -368,6 +409,9 @@ def kepconvert_main():
     parser.add_argument('infile', help='Name of input file', type=str)
     parser.add_argument('conversion', help='Type of data conversion', type=str,
                         choices=['fits2asc', 'fits2csv', 'asc2fits'], default='fits2asc')
+    parser.add_argument('--timeformat', dest='timeformat', default='',
+                        help="Export time into any subformat handled by astropy.Time",

```

Comment:
I suggest adding a few examples to the help string, i.e.: `...any subformat handled by astropy.time (e.g. mjd, iso, decimalyear)`.  It should also say `default="jd"`.